src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
make method non-static

Reply:
I can make that change, but I would appreciate it if you could explain to me why it should be changed. What are the potential problems that could occur if we use a static method here?

Thanks in advance.